apps/process-new-granules/src/process_new_granules.py

```diff
@@ -18,22 +21,37 @@ def get_unprocessed_granules(subscription):
         name=subscription['job_specification']['name'],
         job_type=subscription['job_specification']['job_type'],
     )
-    processed_granules = itertools.chain(*[job['job_parameters']['granules'] for job in processed_jobs])
+    #  TODO This line could be made a lot clearer.
+    processed_granules = itertools.chain(*[job['job_parameters']['granules'][0] for job in processed_jobs])
     return list(set(all_granules) - set(processed_granules))
 
 
+def get_neighbors(granule, depth):
+    return []
```

Comment:
with asf_search 3.0.3 it's something like:
```suggestion
    reference = asf_search.search(granule_list='granule', processingLevel='SLC')[0]
    stack = asf_search.baseline_search.stack_from_product(reference)
    neighbors = [item['sceneName'] for item in neighbors[-depth-1:-1]]
    return neighbors
```